### ISSUE TITLE
Cannot get past "loading your logseq documents" on journals page

### ISSUE THREAD

**Issue by dpnova** [galihlprakoso/logseq-plugin-assistseq-ai-assistant#12]
I have put a gemini api key in but can't get past this screen. Have chosen gemini for embedded and regular and have tried with each of the available gemini models.

![image](https://github.com/user-attachments/assets/bb7b54c5-35d5-49b4-89a4-bf6a51523b30)

dev tools doesn't have any obvious related errors, there's no attempts to make any network requests either

cheers!

**Comment by dpnova** [comment]
Had a quick look in the source - I think the issue here is this currently doesnt work on the journals page.

**Comment by joegoldin** [comment]
> Had a quick look in the source - I think the issue here is this currently doesnt work on the journals page.

Seeing the same behaviour on my end, thanks for the post!

**Comment by fpigeonjr** [comment]
yeah demo looks like its not on the journals page but on an individual page that had several links

**Comment by galihlprakoso** [comment]
> Thanks for this [@galihlprakoso](https://github.com/galihlprakoso) - it looks like it loads on the journal page, but journals aren't included in the context, is that right?

Now when you opening the journals page, it will enter "Global Mode". it will search to the entire documents and use it as context. "e.g show me all of my pending TODO list" something like that @dpnova
